### PR TITLE
Update add-tas-connector.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/connectors/cloud-providers/add-tas-connector.md [hunk 1]
```diff
@@ -190,3 +190,22 @@ Perform the following steps to add a TAS connector.
     
     The connector now appears in the **Connectors** list.
 
+
+### Refresh Token Support
+
+:::note
+
+Currently, Refresh Token Authentication support is behind the feature flag ``. Contact [Harness Support](mailto:support@harness.io) to enable the feature. The delegate version required is `818XX` to leverage this feature.
```
Comment (brian-f-harness) [inline docs/platform/connectors/cloud-providers/add-tas-connector.md:198]:
```suggestion
Currently, Refresh token authentication support is behind the feature flag ``. Contact [Harness Support](mailto:support@harness.io) to enable the feature. Harness Delegate version  23.12.81804 or higher is required for this feature.
```

File: docs/platform/connectors/cloud-providers/add-tas-connector.md [hunk 2]
```diff
@@ -190,3 +190,22 @@ Perform the following steps to add a TAS connector.
...
+
+:::
+
+Harness offers the ability to provide a Refresh Token to the Tanzu Connector. We use this refresh token to authenticate with your Tanzu instance. You must still provide a username and password to authenticate with tanzu. We need that credential to fetch a new refresh token. Once the refresh token is provided in the connector, Harness leverages the token to authenticate to perform each task. When Harness performs each step, Harness will authenticate with the refresh token and then perform the tanzu step defined in the pipeline.
```
Comment (brian-f-harness) [inline docs/platform/connectors/cloud-providers/add-tas-connector.md:202]:
```suggestion
Harness provides the option to use a Refresh token to authenticate with the Tanzu connector. This Refresh token is used by Harness to verify your Tanzu instance. However, you still need to provide a username and password to authenticate with Tanzu. These credentials are used to obtain a new Refresh token. Once the Refresh token is provided in the connector, Harness uses it to authenticate and perform each task. Harness will authenticate with the Refresh token before executing each Tanzu step defined in the pipeline.
```

File: docs/platform/connectors/cloud-providers/add-tas-connector.md [hunk 3]
```diff
@@ -190,3 +190,22 @@ Perform the following steps to add a TAS connector.
...
+
+:::note
+
+Currently, Refresh Token Authentication support is behind the feature flag ``. Contact [Harness Support](mailto:support@harness.io) to enable the feature. The delegate version required is `818XX` to leverage this feature.
+
+:::
+
+Harness offers the ability to provide a Refresh Token to the Tanzu Connector. We use this refresh token to authenticate with your Tanzu instance. You must still provide a username and password to authenticate with tanzu. We need that credential to fetch a new refresh token. Once the refresh token is provided in the connector, Harness leverages the token to authenticate to perform each task. When Harness performs each step, Harness will authenticate with the refresh token and then perform the tanzu step defined in the pipeline. 
+
+You can retrieve the refresh token via the config.json you receive when authenticating with the cf client. You can pass the refresh token as a secret stored in the Harness Secrets Manager or your secrets manager of choice.
```
Comment (brian-f-harness) [inline docs/platform/connectors/cloud-providers/add-tas-connector.md:204]:
```suggestion
You can retrieve the Refresh token via the `config.json` you receive when authenticating with the cf client. You can pass the Refresh token as a secret stored in the Harness Secrets Manager or your secrets manager of choice. 
```

File: docs/platform/connectors/cloud-providers/add-tas-connector.md [hunk 4]
```diff
@@ -190,3 +190,22 @@ Perform the following steps to add a TAS connector.
     
     The connector now appears in the **Connectors** list.
 
+
+### Refresh Token Support
+
+:::note
+
+Currently, Refresh Token Authentication support is behind the feature flag ``. Contact [Harness Support](mailto:support@harness.io) to enable the feature. The delegate version required is `818XX` to leverage this feature.
+
```
Comment (brian-f-harness) [inline docs/platform/connectors/cloud-providers/add-tas-connector.md:199]: @thisrohangupta What is the name of the FF?

Comment (thisrohangupta) [reply docs/platform/connectors/cloud-providers/add-tas-connector.md:199]: CDS_CF_TOKEN_AUTH, i have updated the file new commit.

File: docs/platform/connectors/cloud-providers/add-tas-connector.md [hunk 5]
```diff
@@ -190,3 +190,22 @@ Perform the following steps to add a TAS connector.
     
     The connector now appears in the **Connectors** list.
 
+
+### Refresh Token Support
+
+:::note
+
+Currently, Refresh Token Authentication support is behind the feature flag `CDS_CF_TOKEN_AUTH`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. The delegate version required is `818XX` to leverage this feature.
```
Comment (brian-f-harness) [inline docs/platform/connectors/cloud-providers/add-tas-connector.md:198]:
```suggestion
Currently, Refresh Token Authentication support is behind the feature flag `CDS_CF_TOKEN_AUTH`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. Harness Delegate version 81804 or higher is required to use this feature.
```

File: docs/platform/connectors/cloud-providers/add-tas-connector.md [hunk 6]
```diff
@@ -190,3 +190,22 @@ Perform the following steps to add a TAS connector.
     
     The connector now appears in the **Connectors** list.
 
+
+### Refresh Token Support
+
+:::note
+
+Currently, Refresh Token Authentication support is behind the feature flag `CDS_CF_TOKEN_AUTH`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. Harness Delegate version 81804 or higher is required to use this feature.
```
Comment (brian-f-harness) [inline docs/platform/connectors/cloud-providers/add-tas-connector.md:198]:
```suggestion
Currently, Refresh Token Authentication support is behind the feature flag `CDS_CF_TOKEN_AUTH`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. Harness Delegate version 23.12.81804 or higher is required to use this feature.
```

File: docs/platform/connectors/cloud-providers/add-tas-connector.md [hunk 7]
```diff
@@ -190,3 +190,22 @@ Perform the following steps to add a TAS connector.
     
     The connector now appears in the **Connectors** list.
 
+
+### Refresh Token Support
+
+:::note
+
+Currently, Refresh Token Authentication support is behind the feature flag `CDS_CF_TOKEN_AUTH`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. Harness Delegate version 23.12.81804 or higher is required to use this feature.
```
Comment (brian-f-harness) [inline docs/platform/connectors/cloud-providers/add-tas-connector.md:198]:
```suggestion
Currently, Refresh token authentication support is behind the feature flag `CDS_CF_TOKEN_AUTH`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. Harness Delegate version 23.12.81804 or later is required to use this feature.
```

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**